packages/components/src/Avatar/Avatar.css

```diff
@@ -1,9 +1,7 @@
-:root {
-  --avatar-size: calc(var(--base-unit) * 2);
+.avatar {
+  --avatar-size: 32px;
```

Comment:
I do see some overrides in product around this. be good to double check they still work as expected.